### DISCUSSION TITLE
Getting Livewire encountered corrupt data when trying to hydrate... on second request (change)

### DISCUSSION THREAD

**Discussion by dscheers** [livewire/livewire#5486] [1. Help] [answered]
I'm getting

Livewire encountered corrupt data when trying to hydrate the [hello-world] component. Ensure that the [name, id, data] of the Livewire component wasn't tampered with between requests.

on the second request/change of a simple input text field, copied from the tutorial....

The welcome blade

```
<html>
<head>
    <title>Livewire Test</title>
    @livewireStyles
</head>
<body>
@livewire('hello-world')
@livewireScripts
</body>
</html>

```

The Live wire blade hello-world

```
<html>
<head>
    <title>Livewire Test</title>
</head>
<body>
    <input wire:model.lazy='name' type='text'>
    Hello {{ $name }}
    {{ csrf_field() }}
</body>
</html>
```

And the Livewire component

```
<?php
namespace App\Http\Livewire;
use Livewire\Component;
class HelloWorld extends Component
{
    public string $name = 'Danny';

    public function mount() {
        $name = 'Danny';
    }

    public function render()
    {
        return view('livewire.hello-world');
    }
}

```

Following the tutorial at https://laravel-livewire.com/screencasts/data-binding

So the first time I change the input field and click outside, it changes the text besides the input filed
Second time I change and click out, it throws the error.

Without lazy, the first change in the field is accepted and displayed, if I keep typing, same error.

Have tried a lot of stuff, nothing seems to work.

**Comment by joshhanley** [comment, marked as answer]
@dscheers the issue is probably with your component's view. You have added `<html>` and `<body>` tags there when there shouldn't be as that is in your layout file.

Try changing your hello-world.blade.php file to this:

```blade
<div>
     <input wire:model.lazy='name' type='text'>
    Hello {{ $name }}
</div>
```

Few things to note, always make sure you have one root element, notice I surrounded everything with a `<div>`. And the other is the CRSF token isn't need with Livewire as Livewire already handles it for you.

Hope this helps!

**Reply by dscheers** [reply]
Works now, thanks a million

**Reply by joshhanley** [reply]
@dscheers no worries! 🙂

**Comment by mhgolij** [comment]
This issue can have various causes, one of which I discussed in the link below:

https://mhgolij.medium.com/what-is-livewire-encountered-corrupt-data-when-trying-to-hydrate-a-component-4e1099ac2f4d
